### PR TITLE
Removed unnecessary update queries for EVs for loss conversion

### PR DESCRIPTION
**Pull request by RobTerwel**
As spotted by @jorisberkhout , the EV efficiency improvement input statement updates the loss output of `transport_car_using_electricity` and `transport_truck_using_electricity`. Since they have the attribute
``` 
output.loss = elastic
```

manipulating it with some numbers does not have any effect.

### CODE REVIEW

**Comment by jorisberkhout** [comment]
:+1:
